src/bilder/images/edxapp/templates/common_values.yml

```diff
@@ -377,8 +378,14 @@ PLATFORM_TWITTER_ACCOUNT: '@YourPlatformTwitterAccount'
 POLICY_CHANGE_GRADES_ROUTING_KEY: edx.lms.core.default
 PRESS_EMAIL: support@mitxonline.mit.edu # MODIFIED
 PROCTORING_BACKENDS:
-    DEFAULT: 'null'
-    'null': {}
```

Comment:
We should keep this `null` backend definition as well so that course teams have the option of setting it as their proctoring backend on a per-course basis.